content/20.results.md

```diff
@@ -18,41 +17,18 @@ For each article, we extracted its last author's fore name and last name for ana
 
 Defining an appropriate background distribution is key to determining the extent to which representation diverges from that background.
 We were faced with multiple decisions as we sought to define a field-specific distribution.
```

Comment:
I think we should talk about these key decisions:
* Honorees vs honors, authors vs authorships? We chose honors/authorships and why.
* What is the background? All of science, journals, MESH? We chose MESH and why.
* Which authors are the senior ones eligible for authorship? Corresponding, last? We chose Last and why.
* How should we evaluate articles? As equal contributions? Weighted by citation? We chose equal and why.

In this section we would reason through each of the decisions without giving information about examining multiple options. Then as the final part, we could say that in multiple versions we evaluated different decisions (refer to methods) and results were consistent (refer to supplement).

Reply:
Then perhaps the title could be: "Defining an evaluation set and field-specific background"

Reply:
I made three of the four points in 551e5a7 and 876cb72. 
- Discussing equal weight on articles is difficult without examining the other option of weighting by citation. I find it really hard to write, especially in the new regression framework we have.